libs/wallet/src/vega-order-transaction-dialog/vega-order-transaction-dialog.tsx

```diff
@@ -108,29 +110,42 @@ export const VegaOrderTransactionDialog = ({
 
   return (
     <OrderDialogWrapper title={title} icon={<Icon name="tick" size={20} />}>
-      <p>{t(`Status: ${finalizedOrder.status}`)}</p>
-      {finalizedOrder.market && (
-        <p>{t(`Market: ${finalizedOrder.market.name}`)}</p>
-      )}
-      <p>{t(`Type: ${finalizedOrder.type}`)}</p>
-      <p>
-        {t(
-          `Amount: ${addDecimal(
-            finalizedOrder.size,
-            finalizedOrder.market?.positionDecimalPlaces || 0
-          )}`
+      <div className="grid grid-cols-1 md:grid-cols-2 gap-8">
+        {finalizedOrder.market && (
+          <div>
+            <p className={headerClassName}>{t(`Market`)}</p>
+            <p>{t(`${finalizedOrder.market.name}`)}</p>
+          </div>
         )}
-      </p>
-      {finalizedOrder.type === 'Limit' && finalizedOrder.market && (
-        <p>
-          {t(
-            `Price: ${addDecimalsFormatNumber(
-              finalizedOrder.price,
-              finalizedOrder.market.decimalPlaces
-            )}`
-          )}
-        </p>
-      )}
+        <div>
+          <p className={headerClassName}>{t(`Status`)}</p>
+          <p>{t(`${finalizedOrder.status}`)}</p>
+        </div>
+        <div>
+          <p className={headerClassName}>{t(`Amount`)}</p>
+          <p className={finalizedOrder.side}>
+            {t(
+              `${addDecimal(
+                finalizedOrder.size,
+                finalizedOrder.market?.positionDecimalPlaces || 0
+              )}`
+            )}
+          </p>
+        </div>
+        {finalizedOrder.type === 'Limit' && finalizedOrder.market && (
```

Comment:
chore (blocking): can we remove the magic string and use the ordertype enum from types package